### PR TITLE
CXP-811: add documentation about new Product ACLs

### PR DESCRIPTION
**Pull request by tseho**
see https://github.com/akeneo/pim-community-dev/pull/15446

### CODE REVIEW

File: content/rest-api/permissions.md [hunk 1]
```diff
@@ -47,8 +47,11 @@ Do not give this permission to your UI user role as it makes no sense to give AP
 :::
 
 :::info
-Note that if a role has `Overall Web API` access, then it means that all the users under that role will be able to make requests on products, product models and published products.  
-There is no way to only restrict access to products, except if you are using a 2.x Enterprise Edition. In this case, the EE permissions based on user groups are applied to the API for the products and the published products.
+**Up to the 5.0 version**, users with `Overall Web API` were able to make requests on products and product models without additional permission.
+
+On SaaS and newer versions, new ACLs were introduced: `Create and update products`, `List products` and `Delete products`.
+
+Moreover, in the Enterprise Edition, the [Catalog Permissions](/documentation/permissions.html#catalog-permissions-ee-only) are still applied on the products and product models, on addition to those ACLs.
```
Comment (RiffVentura) [inline content/rest-api/permissions.md:54]:
```suggestion
Moreover, in the Enterprise Edition, the [Catalog Permissions](/documentation/permissions.html#catalog-permissions-ee-only) are still applied on the products and product models, in addition to those ACLs.
```

File: content/rest-api/permissions.md [hunk 2]
```diff
@@ -47,8 +47,11 @@ Do not give this permission to your UI user role as it makes no sense to give AP
 :::
 
 :::info
-Note that if a role has `Overall Web API` access, then it means that all the users under that role will be able to make requests on products, product models and published products.  
-There is no way to only restrict access to products, except if you are using a 2.x Enterprise Edition. In this case, the EE permissions based on user groups are applied to the API for the products and the published products.
+**Up to the 5.0 version**, users with `Overall Web API` were able to make requests on products and product models without additional permission.
```
[the remark this answers is not in the document]
Comment (tseho) [reply content/rest-api/permissions.md:50]: Don't worry, for the english, I will validate with Camille.

[the remark this answers is not in the document]
Comment (camillebourjade) [reply content/rest-api/permissions.md:50]:
```suggestion
**Until the 5.0 version**, users with `Overall Web API` were able to make requests on products and product models without additional permissions.
```

File: content/rest-api/permissions.md (hunk 1 above)
Comment (camillebourjade) [reply content/rest-api/permissions.md:54]: +1

**Review by RiffVentura — APPROVED** [review]

File: content/rest-api/permissions.md [hunk 3]
```diff
@@ -47,8 +47,11 @@ Do not give this permission to your UI user role as it makes no sense to give AP
 :::
 
 :::info
-Note that if a role has `Overall Web API` access, then it means that all the users under that role will be able to make requests on products, product models and published products.  
-There is no way to only restrict access to products, except if you are using a 2.x Enterprise Edition. In this case, the EE permissions based on user groups are applied to the API for the products and the published products.
+**Until the 5.0 version**, users with `Overall Web API` were able to make requests on products and product models without additional permissions.
+
+On SaaS and newer versions, new ACLs were introduced: `Create and update products`, `List products` and `Delete products`.
```
Comment (meganelepalud-akeneo) [inline content/rest-api/permissions.md:52]:
```suggestion
We introduced the following ACLs on **Akeneo SaaS versions** for more security: `Create and update products`, `List products` and `Delete products`.
```

Comment (meganelepalud-akeneo) [reply content/rest-api/permissions.md:52]: I have a proofreading timeslot on Thursday with Camille. I'll check with her if that's ok to change this sentence. I'd like to add the Security argument to explain why we made it.

**Review by meganelepalud-akeneo — APPROVED** [review]

**Pull request merged**